pkg/services/user/userimpl/user.go

```diff
@@ -23,6 +20,8 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 	"github.com/grafana/grafana/pkg/setting"
 	"github.com/grafana/grafana/pkg/util"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
```

Comment:
please revert this change

Reply:
When I ran "go fmt ./..." even on v3 branch this change appear. It is moved from top to here. Same if I save in my IDE.

Reply:
I dont see make format in Makefile. Only something called format-drone. Not related.

Reply:
please revert manually if there is no command for that.